.github/workflows/integration-tests.yml

```diff
@@ -120,27 +106,95 @@ jobs:
             zkvm: zisk
           - test: prove_panic_guest
             zkvm: zisk
+    uses: ./.github/workflows/run-benchmark.yml
+    with:
+      test: ${{ matrix.test }}
+      zkvm: ${{ matrix.zkvm }}
+      el: 'none'
+      threads: 12
 
+  generate-benchmark-website:
```

Comment:
This and next jobs are new ones pulling uploaded artifacts, merging, generating website and publishing.

Pending to only run this on `master` -- for now is always running to test the PR.